code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -646,6 +651,15 @@ GLOBAL_LIST_EMPTY(PDAs)
 			if("MessageAll")
 				send_to_all(U)
 
+			if("toggle_block")
+				toggle_blocking(usr, href_list["target"])
```

Comment:
```
				var/obj/item/pda/P = locate(href_list["target"]) in GLOB.PDAs
				if(P)
					toggle_blocking(usr, P)
```
or
```
				toggle_blocking(usr, locate(href_list["target"]) in GLOB.PDAs)
```
Though, in the latter case, it's still required to check if the target exists or not in the proc itself.

Reply:
EDIT: it uses the owner name as target, I'm dumb and no real issue here.